pkg/logql/expr.y.go

```diff
@@ -1,4 +1,4 @@
-// Code generated by goyacc -p expr -o pkg/logql/expr.go pkg/logql/expr.y. DO NOT EDIT.
+// Code generated by goyacc -p expr -o pkg/logql/expr.y.go pkg/logql/expr.y. DO NOT EDIT.
```

Comment:
is this new ?

Reply:
yeah, following the format of the protobuf files which are `.pb.go` I moved the yacc files to be `.y.go` 

Because the old target for yacc files

```
%.go: %.y
```

Matched every go file being compiled, this way that target is recreated as:

```
%.y.go: %.y
```

And only matches the yacc files